src/wallet/wallet.go

```diff
@@ -99,6 +101,35 @@ type Options struct {
 	CryptoType CryptoType // wallet encryption type, scrypt-chacha20poly1305 or sha256-xor.
 }
 
+type Receiver struct {
+	Address string `json:"address"`
+	Coins   string `json:"coins"`
+	Hours   string `json:"hours"`
```

Comment:
Implement a custom type with MarshalJSON/UnmarshalJSON for Hours and Coins, to parse string to droplets/hours respectively